src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AutoDisenchanter.java

```diff
@@ -129,21 +127,15 @@ private void transferEnchantments(ItemStack item, ItemStack book, Map<Enchantmen
         item.setItemMeta(itemMeta);
         book.setItemMeta(bookMeta);
 
-        EnchantmentStorageMeta meta = (EnchantmentStorageMeta) book.getItemMeta();
-
         for (Map.Entry<Enchantment, Integer> entry : enchantments.entrySet()) {
             item.removeEnchantment(entry.getKey());
-            meta.addStoredEnchant(entry.getKey(), entry.getValue(), true);
         }
 
-        book.setItemMeta(meta);
+        book.addUnsafeEnchantments(enchantments);
```

Comment:
You need the EnchantmentStorageMeta not the book's enchants

Reply:
Reverted to the old ones.